src/client/datascience/types.ts

```diff
@@ -184,7 +173,6 @@ export interface INotebookServerOptions {
     skipUsingDefaultConfig?: boolean;
     workingDir?: string;
     purpose: string;
-    skipSearchingForKernel?: boolean;
```

Comment:
Not used, & rightfully, we don't need to search for kernels when starting a server.